content/05kapitel.tex

```diff
@@ -0,0 +1,10 @@
+% !TeX root = ../dokumentation.tex
+
+\chapter{Projekt Ergebnisse}
```

Comment:
regarding our organisational changes (which you clarified should be mentioned in our section for sprints), could we add a section here elaborating on analysis of their results?

Reply:
Good point omw to change

Reply:
Seeing how you added a couple of commits, which section is this?